beefy-gadget/src/worker.rs

```diff
@@ -101,8 +102,10 @@ where
 	/// BEEFY pallet has been deployed on-chain.
 	///
 	/// The BEEFY pallet is needed in order to keep track of the BEEFY authority set.
+	#[allow(clippy::too_many_arguments)]
```

Comment:
Couldn't this thing use `BeefyParams` as well?

Reply:
Yep, I'm not done yet :-)

Reply:
I have introduced `WorkerParams` as well.